src/util/util.js

```diff
@@ -385,6 +385,28 @@ export function isClosedPolygon(points: Array<Point>): boolean {
     return Math.abs(calculateSignedArea(points)) > 0.01;
 }
 
+/**
+ * Retruns the maximum value of an element in an array,
+ * based on the value retured by the comparator function. Returns null if the
+ * array is empty
+ *
+ * @export
+ * @param {Array<any>} array
+ * @param {(elem: any) => number} comparator
+ * @returns {*}
+ */
+export function max(array: Array<any>, comparator: (elem: any) => number): ?any {
```

Comment:
Where is this function being called?

Reply:
Agree that it doesn't seem used.

Reply:
Ah, yes. I was using a different approach to determine the `outer shell` polygon earlier. Which was picking the polygon with the largest `area`. I later realized that that was necessary since, that always seems to be at `polygons[0]`. I left the util function in there just in case we need it later.

Reply:
I think we should avoid adding code we aren't either:
- using internally
or
- exposing as an API for map developers